packages/webapp/src/containers/InviteSignUp/index.jsx

```diff
@@ -68,37 +73,76 @@ function InviteSignUp({ history }) {
       });
     }
   };
-  const clientId = import.meta.env.VITE_GOOGLE_OAUTH_CLIENT_ID;
+
+  const login = useGoogleLogin({
+    onSuccess: async (tokenResponse) => {
+      try {
+        // I have not previously used redux or redux-saga, but I imagine a query to an API like this should be in a saga.js file?
```

Comment:
@kathyavini 

I might not have noticed this otherwise haha. But I think your instinct is correct.

Reply:
Haha, alas. Ok, I'll do some reading to figure out how to structure that.

Reply:
This does not update any Redux state, so I don't think it needs to be done as a Redux action. I might be wrong though...

Reply:
I think asynchronous requests are usually handled in sagas. For example this would be the first axios request in any non-saga.js file in /containers .